practice-two/src/javascript/main.js

```diff
@@ -1,4 +1,8 @@
 /* global document */
+
 import { renderAllNotes } from './modules/getNotes';
+import { renderNotesByCategory } from './modules/filterNotes';
 
 document.addEventListener('DOMContentLoaded', renderAllNotes);
+
+renderNotesByCategory();
```

Comment:
@quanghuy1799 The function `renderNotesByCategory` requires at least one parameter as a type of category. Why did you call with an empty parameter here?